script/pip_check

```diff
@@ -3,7 +3,7 @@ PIP_CACHE=$1
 
 # Number of existing dependency conflicts
 # Update if a PR resolve one!
-DEPENDENCY_CONFLICTS=6
+DEPENDENCY_CONFLICTS=7
```

Comment:
```suggestion
DEPENDENCY_CONFLICTS=6
```

Reply:
Yup, as soon as https://github.com/home-assistant/core/pull/77631 merges we can do that 👍

Reply:
Once the CI gives 77631 the 👍 , I'll merge this, and apply this.

Then #77576 will need to be rebased for the lib bump, and we can merge it once we get a 👍  from Erik